### PR TITLE
Fixes #458 added privacy policy

### LINKED ISSUE #458: Add privacy policy for uploading app on play store
**Actual Behaviour**

Currently no privacy policy.

**Expected Behaviour**

App needs a privacy policy for uploading it to play store.

**Steps to reproduce it**

NA

**LogCat for the issue**

NA

**Screenshots of the issue**

NA

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by viveksb007**
Fixes issue #458

Changes: 
- Added privacy policy to upload apk on play store

[Template Used](https://gist.github.com/alphamu/c42f6c3fce530ca5e804e672fed70d78)

### CODE REVIEW

File: docs/privacy_policy.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<html>
+<body>
+<h2>Privacy Policy</h2>
+<p>Fossasia built the Pocket Science Lab app as a open source app. This SERVICE is provided by Fossasia  and is intended for use as is.</p>
```
Comment (CloudyPadmal) [inline docs/privacy_policy.html]: I think `FOSSASIA` needs to be in all caps

File: docs/privacy_policy.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    relation with this policy. The Personal Information that we collect are used for providing and
+    improving the Service. We will not use or share your information with anyone except as described
+    in this Privacy Policy.</p>
+<p>The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions,
+    which is accessible at Pocket Science Lab app, unless otherwise defined in this Privacy Policy.</p>
+
+<p><strong>Information Collection and Use</strong></p>
+<p>For a better experience while using our Service, we may require you to provide us with certain
+    personally identifiable information, including but not limited to users name, address, location, pictures. 
+	The information that we request is will be retained by us and used as described in this privacy policy.</p>
```
Comment (CloudyPadmal) [inline docs/privacy_policy.html]: we request ~~is~~ will be

File: docs/privacy_policy.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+<p><strong>Security</strong></p>
+<p>We value your trust in providing us your Personal Information, thus we are striving to use
+    commercially acceptable means of protecting it. But remember that no method of transmission over
+    the internet, or method of electronic storage is 100% secure and reliable, and we cannot
+    guarantee its absolute security.</p>
+
+<p><strong>Links to Other Sites</strong></p>
+<p>This Service may contain links to other sites. If you click on a third-party link, you will be
+    directed to that site. Note that these external sites are not operated by us. Therefore, I
```
Comment (CloudyPadmal) [inline docs/privacy_policy.html]: Therefore, ~~I~~ We

**Comment by viveksb007** [comment]
I have made the requested changes.
Thanks for reviewing @CloudyPadmal

**Pull request merged**